Fix text outline for home page title and subtitle

miqid opened the pull request:
Addresses #2.

@nii236, I used the existing "Arbutus Slab" font to recreate the home page title and subtitle in an SVG format. Although it's a little heavy handed, I think it's less tedious than fidgeting with the CSS and seeing how the resulting text displays across different browsers.

I don't have the environment set up on my PC to get Hugo regenerating a site, so I'm not able to preview if my changes are good. However, in theory they should display OK. I took the liberty of manually adjusting the generated site index page too, but please try regenerating just to see if anything breaks, since I may have missed updating something specific to how Hugo works.

ghost commented:
Amended changes to target files in the `static` directory now that 2566d1f makes it available.

I've also removed all changes that target items in the `public` directory.

miqh committed:
Replace home page title texts with PNG representations

- Adds PNG representations of both the title and description
  text as they were on the home page
- Updates relevant HTML/CSS to use the new PNG files.

ghost commented:
Fidgeting with the SVG files were proving to be a hassle so I ended up creating PNG representations of the title and description text instead (not to mention the PNG files were considerably smaller in size). Whacked in some extra measures in CSS to make the PNG images scale a little in case the view port width gets too tight.

nii236 commented:
:+1: Will regenerate and see how we go
